src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -14,37 +14,39 @@
 
 import logging
 import math
-
 import requests
-from typing import Mapping, Optional, Sequence
-
-from opentelemetry.sdk._metrics.export import (
-    MetricExporter,
-    Metric,
-    MetricExportResult,
-)
+from typing import Mapping, Optional
 
 from dynatrace.metric.utils import (
     DynatraceMetricsSerializer,
     DynatraceMetricsApiConstants,
     DynatraceMetricsFactory,
     MetricError
 )
-from opentelemetry.sdk._metrics.point import (
+
+from opentelemetry.sdk.metrics.export import (
+    MetricExporter,
+    MetricExportResult,
     Sum,
     AggregationTemporality,
     Gauge,
-    Histogram)
+    Histogram,
+    MetricsData,
+    DataPointT,
+    NumberDataPoint,
+    Metric,
+    HistogramDataPoint
+)
 
 VERSION = "0.3.0b0"
 
 
-def _get_histogram_max(histogram: Histogram):
+def _get_histogram_max(histogram: HistogramDataPoint):
```

Comment:
I am aware of this, but I think extracting a method would hurt readability, as the current code needs to interrupt the execution flow. 

However, since `HistogramDataPoint` now has a `count` field, I used that to reduce the duplicated code to 13 lines (6 comments, one empty) in [e1795a2](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/e1795a2547b0a21ee1e0d219a0dc73f3f6316bbf) :slightly_smiling_face: